replicator-security/docker-compose_dest_sasl_plain_auth.yml

```diff
@@ -101,53 +63,7 @@ services:
       KAFKA_INTER_BROKER_LISTENER_NAME: SASL_PLAINTEXT
       KAFKA_ADVERTISED_LISTENERS: SASL_PLAINTEXT://destKafka1:11091
       KAFKA_BROKER_ID: 1
-      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 3
-      KAFKA_DELETE_TOPIC_ENABLE: "true"
-      KAFKA_AUTO_CREATE_TOPICS_ENABLE: "false"
-      KAFKA_SASL_ENABLED_MECHANISMS: PLAIN
-      KAFKA_SASL_MECHANISM_INTER_BROKER_PROTOCOL: PLAIN
-      KAFKA_OPTS: -Djava.security.auth.login.config=/etc/kafka/secrets/broker_jaas.conf
-  destKafka2:
-    image: confluentinc/cp-enterprise-kafka:5.3.1
-    hostname: destKafka2
-    container_name: destKafka2
-    cpus: 0.3
-    depends_on:
-      - destZookeeper
-    ports:
-      - "11092:11092"
-    volumes:
-      - $PWD/scripts/security:/etc/kafka/secrets
-    environment:
-      KAFKA_ZOOKEEPER_CONNECT: "destZookeeper:2281"
-      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: SASL_PLAINTEXT:SASL_PLAINTEXT
-      KAFKA_INTER_BROKER_LISTENER_NAME: SASL_PLAINTEXT
-      KAFKA_ADVERTISED_LISTENERS: SASL_PLAINTEXT://destKafka2:11092
-      KAFKA_BROKER_ID: 2
-      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 3
-      KAFKA_DELETE_TOPIC_ENABLE: "true"
-      KAFKA_AUTO_CREATE_TOPICS_ENABLE: "false"
-      KAFKA_SASL_ENABLED_MECHANISMS: PLAIN
-      KAFKA_SASL_MECHANISM_INTER_BROKER_PROTOCOL: PLAIN
-      KAFKA_OPTS: -Djava.security.auth.login.config=/etc/kafka/secrets/broker_jaas.conf
-  destKafka3:
-    image: confluentinc/cp-enterprise-kafka:5.3.1
-    hostname: destKafka3
-    container_name: destKafka3
-    cpus: 0.3
-    depends_on:
-      - destZookeeper
-    ports:
-      - "11093:11093"
-    volumes:
-      - $PWD/scripts/security:/etc/kafka/secrets
-    environment:
-      KAFKA_ZOOKEEPER_CONNECT: "destZookeeper:2281"
-      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: SASL_PLAINTEXT:SASL_PLAINTEXT
-      KAFKA_INTER_BROKER_LISTENER_NAME: SASL_PLAINTEXT
-      KAFKA_ADVERTISED_LISTENERS: SASL_PLAINTEXT://destKafka3:11093
-      KAFKA_BROKER_ID: 3
-      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 3
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
```

Comment:
Based on our conversation on Friday, I thought you wanted to keep destination to replication 3 because these examples were used in production by users?

Reply:
I tried it and it still blew up performance on my Mac (in fact it got really bad). The only downside to dropping to 1 is that we now need to configure confluent.topic.replication.factor so I think it still gives an uncluttered view of Replicator security.